extensions/ql-vscode/gulpfile.ts/webpack.config.ts

```diff
@@ -55,6 +55,19 @@ export const config: webpack.Configuration = {
             loader: 'css-loader'
           }
         ]
+      },
+      {
+        test: /\.(woff(2)?|ttf|eot|svg)$/,
+        use: [
+          {
+            loader: 'file-loader',
+            options: {
+              name: '[name].[ext]',
+              outputPath: 'fonts/',
```

Comment:
It doesn't feel right to be putting svg files in `fonts/` to me 🤔 do we need svgs for codicons or can we leave that out for now?

Reply:
We can leave that out for now, this just includes all possible font extension types. However, Codicons doesn't use SVG so we can leave it out.